engine-tests/src/tests/one_inch.rs

```diff
@@ -98,8 +103,8 @@ fn test_1_inch_limit_order_deploy() {
         SubmitResult::try_from_slice(&outcome.return_data.as_value().unwrap()).unwrap();
     assert!(result.status.is_ok());
 
-    // more than 4 million Ethereum gas used
-    assert!(result.gas_used > 4_000_000);
+    // more than 3.5 million Ethereum gas used
+    assert!(result.gas_used > 3_500_000);
```

Comment:
I don't understand why this change was needed; CI was failing on this assert, but I could not reproduce it locally. Maybe a difference in the solidity compiler versions? But I don't know why this just started failing now.

Reply:
I'll verify.

Reply:
I can reproduce the issue locally if I totally clear the `target` folder, forcing a re-download of the 1inch source code. Looks like they [merged a PR yesterday](https://github.com/1inch/limit-order-protocol/pull/47) to lower their EVM gas usage, so it makes sense that we need to lower this now.

I think we should merge this for now, but we can think about pinning a specific commit of the 1inch source code for this test in a future PR.